source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: "WeatherFlow Local"
+description: "Instructions on how to integrate your WeatherFlow tempest into Home Assistant."
+ha_release: 2023.2.0
+ha_category: 
+  - Sensor
+ha_platforms:
+ - sensor
+ - enviornment
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@natekspencer'
+  - '@jeeftor'
+ha_domain: weatherflow
+---
+
+## Overview
+
+This integration is a local-only WeatherFlow integration that reads UDP data from a [WeatherFlow Tempest](https://weatherflow.com/tempest-weather-system/) compatible weather stations. It utilizes WeatherFlow's published [UDP API](https://weatherflow.github.io/Tempest/api/udp.html) to decode and present the data.
+
+## Installation Notes
+
+When you add this integration to Home Assistant it will detect and decode any valid UDP packets sent from a valid weather station. Sensors will only be created in Home Assistant once it has received a valid reading for that particular sensor type.
```

Comment:
What does this even mean? Why would an end-user care about this?
Can we make this more clear, e.g., for my dad?

Reply:
Agreed, I don't think the installation notes is necessary.